src/app/products/products-mix/create-product-mix/create-product-mix.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="productMixForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Select product</mat-label>
+            <mat-select required formControlName="productId">
+              <mat-option *ngFor="let productOptions of productsOptionsData" [value]="productOptions.id">
+                {{ productOptions.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="productMixForm.controls.productId.hasError('required')">
+              Select product is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field *ngIf="this.productsData">
```

Comment:
I don't think restrictedProductData would be a good name for the variable since it doesn't only contain restricted products, it contains both allowed and restricted products.